AstarteDeviceSDKCSharp/Device/AstarteDevice.cs

```diff
@@ -226,7 +226,7 @@ public async Task Connect()
             }
             catch (AstarteCryptoException)
             {
-                Trace.WriteLine("Regenerating the cert");
+                AstarteLogger.Info("Regenerating the cert", this.GetType().Name);
```

Comment:
Keep this type of login only  in Debug mode
```suggestion
                AstarteLogger.Debug("Regenerating the cert", this.GetType().Name);
```